webapp/home/subdomains/genome/content/annotation.py

```diff
@@ -117,7 +117,7 @@
         "button_link": "https://usegalaxy.org.au/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fjbrowse%2Fjbrowse",
     },
     {
-        "title_html": '<code>Prokka</code>',
+        "title_html": '<code>Prokka</code>' - 'Genome annotation, prokaryotes only',
```

Comment:
Should be `'<code>Prokka</code> - Genome annotation, prokaryotes only'`.

Reply:
Not two different strings

Reply:
fixed! I think. I re-pushed so I think it has updated the PR.